src/node_contextify.cc

```diff
@@ -1611,7 +1611,9 @@ static std::vector<std::string_view> throws_only_in_cjs_error_messages = {
     "Identifier '__filename' has already been declared",
     "Identifier '__dirname' has already been declared",
     "await is only valid in async functions and "
-    "the top level bodies of modules"};
+    "the top level bodies of modules",
+    "Identifier 'require' has already been declared",
```

Comment:
`"Identifier 'require' has already been declared"` is already in this list (look five lines up, on line 1610).